### PR TITLE
[cherry-pick] Update README.md (#8281)

### PR DESCRIPTION
**Pull request by ManfeiBai**
[cherry-pick] Update README.md (#8281)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,12 +154,12 @@ GPU and nightly builds are available in our public GCS bucket.
 
 | Version | Cloud GPU VM Wheels |
 | --- | ----------- |
-| 2.5 (CUDA 12.1 + Python 3.9) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp39-cp39-linux_x86_64.whl` |
-| 2.5 (CUDA 12.1 + Python 3.10) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp310-cp310-linux_x86_64.whl` |
-| 2.5 (CUDA 12.1 + Python 3.11) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp311-cp311-linux_x86_64.whl` |
-| 2.5 (CUDA 12.4 + Python 3.9) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.4/torch_xla-2.5.0-cp39-cp39-linux_x86_64.whl` |
-| 2.5 (CUDA 12.4 + Python 3.10) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.4/torch_xla-2.5.0-cp310-cp310-linux_x86_64.whl` |
-| 2.5 (CUDA 12.4 + Python 3.11) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.4/torch_xla-2.5.0-cp311-cp311-linux_x86_64.whl` |
+| 2.5 (CUDA 12.1 + Python 3.9) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp39-cp39-manylinux_2_28_x86_64.whl` |
```
Comment (tengyifei) [inline README.md:157]: Is the "manylinux" intended? Previously it's "linux".

**Review by tengyifei — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ManfeiBai) [reply README.md:157]: yes, this wheel name change after https://github.com/pytorch/xla/pull/8268

File: README.md [hunk 2]
```diff
@@ -203,9 +203,13 @@ The torch wheel version `2.5.0.dev20240820+cpu` can be found at https://download
 
 | Version | GPU Wheel |
 | --- | ----------- |
+| 2.5 (CUDA 12.1 + Python 3.9) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp39-cp39-manylinux_2_28_x86_64.whl` |
 | 2.5 (CUDA 12.1 + Python 3.10) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp310-cp310-manylinux_2_28_x86_64.whl` |
 | 2.5 (CUDA 12.1 + Python 3.11) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp311-cp311-manylinux_2_28_x86_64.whl` |
-| 2.4 (CUDA 12.1 + Python 3.8) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.4.0-cp38-cp38-manylinux_2_28_x86_64.whl` |
```
Comment (zpcore) [inline README.md:208]: Is this one deprecated?

Comment (ManfeiBai) [reply README.md:208]: this wheel don't exits, and we don't have trigger for this too: [link](https://github.com/pytorch/xla/blob/6d690904dc92e16a3041e495421f177575a9f4aa/infra/tpu-pytorch-releases/artifacts.auto.tfvars#L118C1-L199C5)

**Pull request merged**